src/components/button/button.scss

```diff
@@ -0,0 +1,102 @@
+.btn {
+  font-size: 1.03rem;
+  padding: 8px 30px;
+  font-weight: 600;
+  color: white;
+  border: 2px solid black;
+
+  &:hover {
+    cursor: pointer;
+  }
+
+  &.primary {
+    background-color: rgb(137, 109, 250);
```

Comment:
Let's store these colors (primary, secondary, success, etc) in scss variables in a `src/styles/colors.scss` file.

I suspect we will need to reuse these colors at some point, so it will be a good idea to do this as to not duplicate the color values in the future.